src/panels/lovelace/elements/hui-state-label-element.ts

```diff
@@ -59,9 +59,12 @@ class HuiStateLabelElement extends LitElement implements LovelaceElement {
     return html`
       <div
         .title="${computeTooltip(this.hass, this._config)}"
-        @ha-click="${this._handleTap}"
-        @ha-hold="${this._handleHold}"
-        .longPress="${longPress()}"
+        @ha-click=${this._handleClick}
+        @ha-hold=${this._handleHold}
+        @ha-dblclick=${this._handleDblClick}
+        .hasDblClick=${this._config!.dbltap_action &&
+          this._config!.dbltap_action!.action !== "none"}
+        .longPress=${longPress()}
```

Comment:
```suggestion
        .longPress=${longPress({ hasDblClick: this._config!.dbltap_action &&
          this._config!.dbltap_action!.action !== "none" })}
```

Reply:
I don't understand how this is supposed to work? This would just set `longPress` to true

Reply:
or set a private via this?

Reply:
something like?
```
export const longPress = directive((options) => (part: PropertyPart) => {
  (part.committer.element as LongPressElement).hasDblClick = options.hasDblClick;
  longPressBind(part.committer.element);
});
```

Reply:
No, we should pass it to `bind` and then check if there:
```
export const longPress = directive((options) => (part: PropertyPart) => {
  longPressBind(part.committer.element, options);
});
```

```
export const longPressBind = (element: LongPressElement) => {
  ...
  longpress.bind(element, options);
};
```

```
public bind(element: LongPressElement, options) {
  ...
    const clickEnd = (ev: Event) => {
      ...
      } else if (options.hasDblClick) {
         ...
      }
}
```